cleanlab/filter.py

```diff
@@ -330,137 +326,169 @@ def get_noise_indices(
       Set this to 1 to REMOVE parallel processing (if its causing issues).
 
     verbose : int
-      If 0, no print statements. If 1, prints when multiprocessing happens."""
+      If 0, no print statements. If 1, prints when multiprocessing happens.
```

Comment:
would be nice to print the number of label issues found if `verbose = 1`

Reply:
added

```
    if verbose:
        print('Number of label issues found: {}'.format(sum(label_issues_mask)))
```